vignettes/release_strategy.Rmd

```diff
@@ -39,34 +39,58 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 # Package Release Process
 
-## Quarterly Release
-
-A package release is done in four parts:
-
-1) Verify that all CI/CD checks and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
+## Planned Release
```

Comment:
There are many other steps we should be checking before a release that are enumerated in the GitHub Release Issue created with `usethis::use_release_issue()`. This includes running R CMD Check on the CRAN Windows devel env, running the CRAN-compliant URL checker (we use the lychee too, which has some different requirements from CRAN), updating CRAN comments, polishing the NEWS file, running reverse dependency checks, etc.

Reply:
I am not sure if we need the issue created by `use_release_issue()`. Looks like a repetition of the items we have here. I would prefer to keep it here (and not duplicate it in `release_bullets()` functions). Then the information is always available and maintained in one place.

If we need any additional checks, I would rather add them to our workflows than create new items in the list. However, I think our workflows are pretty good. In my experience, the code was ready for submission if our workflows had passed. Only sometimes some links were missed.

Reply:
oohhh i like the release issue and the abilities to add additional items, e.g. the readme items for presentations, schedule that Edoardo has worked on.  

i already made one https://github.com/pharmaverse/admiral/issues/2256

Every release is pretty annoying honestly.  maybe some of these extra might help us.

Reply:
I find the release issue very helpful even after having done many many releases to CRAN. 

But since it's Ben's burden, we should go with what's best for him as the maintainer.

Reply:
Hm, maybe admiral is different. For admiralonco releasing was quite easy so far. I have added a paragraph mentioning `use_release_issue()`.

Reply:
yeah every time there is some small issue from CRAN.  then the bugs! actually...can roche start checking the package now rather than after it gets to CRAN as you all's process surfaces some issues

Reply:
I think checking admiral on company level before the release could avoid some patch release. We would need to change our release process for this (and most likely our branching strategy). At the moment we are updating the release candidate until a few days before the release. So we do not have enough time to update the company-specific code and test it with the new version. We would need to add a period between freezing the release candidate and submitting to CRAN.

Reply:
it makes sense that admiralonco is an easier release because admiral has reverse dependencies, more users, and is a complex package in many ways.